cypress/integration/page.spec.js

```diff
@@ -819,13 +876,15 @@
       });
 
       it('should not show NewCommentForm', () => {
-        newCommentForm.el().should('not.exist')
+        newCommentForm.el().should('not.exist');
       });
     });
 
     describe('after empty comments received', () => {
       beforeEach(() => {
-        cy.intercept('**/comments?postId=1', { body: [] }).as('post1CommentsRequest');
+        cy.intercept('**/comments?postId=1', { body: [] }).as(
+          'post1CommentsRequest',
+        );
```

Comment:
The `cy.intercept` call is missing a semicolon at the end. Ensure that each statement is properly terminated to avoid syntax errors.